src/main/kotlin/com.bugsnag.android.gradle/AndroidManifestParser.kt

```diff
@@ -63,6 +62,8 @@ class AndroidManifestParser {
                     |versionCode=$versionCode
                     |versionName=$versionName
                     |applicationId=$applicationId
+                    |Manifest file = $manifestPath
+                    |Manifest content=${manifestPath.readText()}
```

Comment:
I accidentally left this in from debugging, but kind of think at least printing the file is useful for debugging in general too. Will remove the content print, let me know if you want to remove the file path too

```suggestion
```